schema/deploy/todos.sql

```diff
@@ -0,0 +1,16 @@
+-- Deploy todo_app:todos to pg
+-- requires: todo_appschema
+
+BEGIN;
+
+set client_min_messages = 'warning';
+
+create table todo_app.todos (
+    id   integer primary key generated always as identity,
+    task  text        not null,
+    completed boolean   not null default false,
+    date_created    timestamptz not null default now(),
+    date_updated    timestamptz not null default now()
```

Comment:
I originally forgot to include `default now()` and to get it in, I had to revert and then redeploy. Is that the right way to do it?

Reply:
In your development environment, yes. For some changes the way might even be to drop and recreate your db.

If this happens in a prod environment, you will have to create a new change which alters the column. The changes that are already deployed should be immutable. You can't revert and redeploy there otherwise you're dropping the tables and loosing your data